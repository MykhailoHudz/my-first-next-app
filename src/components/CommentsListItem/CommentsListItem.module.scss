.list_number {
  font-weight: 700;
}
