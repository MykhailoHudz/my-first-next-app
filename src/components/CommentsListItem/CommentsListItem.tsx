import styles from "./CommentsListItem.module.scss";

function CommentsListItem({ id, body }: { id: number; body: string }) {
  return (
    <li>
      <p>
        <span className={styles.list_number}>{id}.&nbsp;</span>
        {body.slice(0, 150) + "..."}
      </p>
    </li>
  );
}

export default CommentsListItem;
