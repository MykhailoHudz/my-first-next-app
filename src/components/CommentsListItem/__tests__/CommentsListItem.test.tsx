import { render, fireEvent, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import CommentsListItem from "../CommentsListItem";

test("matches snapshot", () => {
  const commentProps = {
    id: 131,
    body: "rem magnam at voluptatem\naspernatur et et nostrum rerum\ndignissimos eum quibusdam\noptio quod dolores et",
  };
  const { asFragment } = render(<CommentsListItem {...commentProps} />);

  expect(asFragment()).toMatchSnapshot();
});
