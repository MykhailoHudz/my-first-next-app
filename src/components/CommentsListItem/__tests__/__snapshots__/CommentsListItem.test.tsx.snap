// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <li>
    <p>
      <span
        class="list_number"
      >
        131.Â 
      </span>
      rem magnam at voluptatem
aspernatur et et nostrum rerum
dignissimos eum quibusdam
optio quod dolores et...
    </p>
  </li>
</DocumentFragment>
`;
